index.js

```diff
@@ -384,7 +384,78 @@ Color.prototype = {
 				w1 * color1.green() + w2 * color2.green(),
 				w1 * color1.blue() + w2 * color2.blue(),
 				color1.alpha() * p + color2.alpha() * (1 - p));
+	},
+
+	gradientRGB: function (toColor, steps) {
+		if (!toColor || !toColor.rgb) {
+			throw new Error('Argument "toColor" was not a Color instance, but rather an instance of ' + typeof toColor);
+		}
+		if (steps < 2) {
+			throw new Error('Argument "steps" must to be bigger or equal to 2.');
+		}
+		toColor = toColor.rgb();
+		var fromColor = this.rgb();
+		var [fR, fG, fB] = fromColor.color;
+		var [tR, tG, tB] = toColor.color;
+		var fA = fromColor.alpha();
+		var tA = toColor.alpha();
+		var incR = (tR - fR) / (steps - 1);
+		var incG = (tG - fG) / (steps - 1);
+		var incB = (tB - fB) / (steps - 1);
+		var incA = (tA - fA) / (steps - 1);
+		var gradient = [fromColor];
+		for (var i = 1; i < steps; i++) {
+			gradient.push(Color({
+				r: Math.round(fR + incR * i),
+				g: Math.round(fG + incG * i),
+				b: Math.round(fB + incB * i),
+				alpha: fA + incA * i
+			}));
+		}
+		return gradient;
+	},
+
+	gradientHSL: function (toColor, steps, rotationWay) {
+		if (!toColor || !toColor.hsl) {
+			throw new Error('Argument "toColor" was not a Color instance, but rather an instance of ' + typeof toColor);
+		}
+		if (steps < 2) {
+			throw new Error('Argument "steps" must to be bigger or equal to 2.');
+		}
```

Comment:
Could introduce a base case for `steps === 2` and return `[this, toColor]`. For `steps === 3`, you could even do `[this, this.mix(toColor, 0.5), toColor]` since it's faster.

Reply:
Ok. I'll do it.